logstash_async/handler.py

```diff
@@ -63,20 +60,14 @@ def emit(self, record):
             return  # we should not do anything, so just leave
 
         self._setup_transport()
-        self._start_worker_thread()
 
         # basically same implementation as in logging.handlers.SocketHandler.emit()
         try:
             data = self._format_record(record)
-            AsynchronousLogstashHandler._worker_thread.enqueue_event(data)
+            self._transport.send([data], use_logging=False)
```

Comment:
I wonder if we should do some more extensive error handling here like we do in https://github.com/eht16/python-logstash-async/blob/master/logstash_async/worker.py#L216 (in terms of filtering common network errors or rate limiting).
But maybe it is actually not relevant for synchronous processing.

Reply:
Hm synchronous handling and then retrying .. would probably delay the request too long and be counterproductive. Yet thats just my thought and I follow your judgement here.

Reply:
I was rather thinking about ignoring stacktraces for common network errors like "connection refused" or timeouts.
But I didn't make up my mind about this and I guess for now it's fine to keep the default behavior.